rpcd/playbooks/roles/kibana/tasks/kibana_post_install.yml

```diff
@@ -40,8 +74,8 @@
   template:
     src: "{{ item }}"
     dest: "/etc/apache2/sites-available/{{ item }}"
-    owner: "{{ kibana_system_user }}"
-    group: "{{ kibana_system_group }}"
+    owner: "root"
+    group: "root"
```

Comment:
Shouldn't these be the new `{{ kibana_service_user }}` and `{{ kibana_service_group }}` variables defined in `main.yml`?